web/src/components/Input.tsx

```diff
@@ -6,9 +6,13 @@ import Theme, { belowBreakpoint } from '../theme'
 export default function Input({
   placeholder,
   className,
+  value,
+  onChange,
 }: {
   placeholder?: string
   className?: string
+  value: string
+  onChange?: (arg0: string) => void
```

Comment:
I'm legit just curious: why not call this `e`?
```suggestion
  onChange?: (e: string) => void
```

Reply:
I guess personal taste